charts/fluid/fluid/templates/webhook/MutatingWebhookConfiguration.yaml

```diff
@@ -21,5 +21,5 @@ webhooks:
     failurePolicy: Ignore
     namespaceSelector:
       matchLabels:
-        Fluid-Injection: enabled
+        fluid.io/inject-fluid-affinity-strategy: "true"
```

Comment:
How about `fluid.io/inject-fluid-affinity-strategy` to `fluid.io/enable-scheduling-strategy` ?

Reply:
how about fluid.io/enable-scheduling-strategy-optimization？

Reply:
fluid.io/enable-scheduling-strategy
done